### PR TITLE
Fix AK streams doc nits

### PR DESCRIPTION
**Pull request by joel-hamill**
- Table formatting in /developer-guide/dsl-api
- Missing /streams/developer-guide `include virtual` files
- Improperly formatted Notes

### CODE REVIEW

**Comment by joel-hamill** [comment]
@guozhangwang

File: docs/documentation/streams/developer-guide/app-reset-tool.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!--
```
Comment (joel-hamill) [inline docs/documentation/streams/developer-guide/app-reset-tool.html:1]: @guozhangwang i assume these files need to be here as well?

**Comment by guozhangwang** [comment]
ping @derrickdoo @miguno for second look.

**Comment by guozhangwang** [comment]
ping again: @derrickdoo @miguno for second look.

**Comment by derrickdoo** [comment]
👍  LGTM

**Comment by guozhangwang** [comment]
retest this please

**Comment by guozhangwang** [comment]
@joel-hamill Does `trunk` need to cherry-pick this PR as well?

**Comment by guozhangwang** [comment]
retest this please

**Review by miguno — APPROVED** [review]

**Comment by guozhangwang** [comment]
Cherry-picked to `trunk` as well. Thanks @joel-hamill !

**Pull request merged**